tests/test_micro.py

```diff
@@ -0,0 +1,362 @@
+import asyncio
+import json
+import random
+
+import nats
+from nats import micro
+
+import unittest
+from unittest import TestCase
+from nats.micro import Verb
+from tests.utils import SingleServerTestCase, async_test
+
+
+class ServiceTest(SingleServerTestCase):
+
+    @async_test
+    async def test_service_basics(self):
+        nc = await nats.connect()
+
+        async def do_add(req):
+            await req.respond(b"42")
+
+        svcs = []
+
+        # Create 5 service responders.
+        for i in range(5):
+            svc = micro.add_service(
+                nc,
+                name="CoolAddService",
+                version="0.1.0",
+                description="Add things together",
+                metadata={"basic": "metadata"},
+            )
+
+            await svc.add_endpoint(
+                name="svc.add",
```

Comment:
Running nats cli version 0.1.4:
```
nats --version
v0.1.4
```

Returns the following error on `nats micro list`:
```
nats: error: /endpoints/0/name: does not match pattern '^[a-zA-Z0-9_-]+$'
```

The endpoint `name` can not have a dot "."